design/one-pager-stacks-security-isolation.md

```diff
@@ -241,13 +235,134 @@ permissionScope: Cluster
 
 ### Stack Install
 
-Since cluster stacks and namespace stacks have differing scopes of execution in the control plane, it will be wise to allow differing access to request their installation.
-A simple way to approach this would be to create different installation request CRDs for each and then use RBAC to restrict access to them by different roles.
+Since cluster stacks and namespace stacks have differing scopes of execution in the control plane, it will be wise to
+ allow differing access to request their installation. A simple way to approach this would be to create different
+ installation request CRDs for each and then use RBAC to restrict access to them by different roles.
+
 Currently, we use a single `StackRequest` type, but going forward we should differentiate by stack type:
 
-* `ClusterStackInstall`: requests the installation of a cluster stack
-* `StackInstall`: requests the installation of a namespace stack
+* `ClusterStackInstall`: requests the installation of a cluster stack who's resources are all cluster scoped
+* `StackInstall`: requests the installation of a namespace stack who's resources are all namespace scoped
+
+With these two separate types, an administrator can allow access to installing cluster stacks to only certain restricted
+ roles, then perhaps be more permissive with access to installing namespaced stacks.
+
+If the install scope type does not match what the stack author has declared within their stack metadata, an error status
+ should be set on the install resource and installation of the stack should not proceed.
+
+### RBAC
+
+We will rely on RBAC to further define a clear separation of concerns for infrastructure resources, and workspace
+ resources in a namespace. In our model we assume that any Cluster Stack resources as they are global will be for the
```

Comment:
perhaps it's simpler to say that Crossplane will provide `Role` and `ClusterRole` definitions for environment and workspace roles (admin, editor, viewer) to simplify RBAC configuration with user-defined `User`, `Group`, `RoleBinding` and `ClusterRoleBinding` resources.